develop.md

```diff
@@ -14,40 +14,24 @@ nextcloud/all-in-one:develop
 And you are done :)
 It will now also select the developer channel for all other containers automatically.
 
-## How to promote builds from develop to latest
-
-<!---
-You can use the Docker CLI to promote builds from develop to latest. Make sure to adjust:
+## How to build new containers
 
-- $name
-- $digest
+Go to https://github.com/nextcloud-releases/all-in-one/actions/workflows/repo-sync.yml and run the workflow that will first sync the repo and then build new container that automatically get published to `develop` and `develop-arm64`.
```

Comment:
Just out of curiosity, is there a reason for the separate tags? It should be possible to build an image for multiple architectures using the same tag with https://docs.docker.com/buildx/working-with-buildx/

Reply:
Yes, there is one :)

See https://github.com/nextcloud/all-in-one/pull/42#issuecomment-986057315

Reply:
Ah I see, thanks for the quick reply :)

Reply:
no problem! Thanks for your reviews! :)